README.md

```diff
@@ -194,10 +194,28 @@ We can then use this method to create a listener that listens for an invitation
 
 When we receive a call we will receive an InviteResponse data class that contains the details we need to accept the call. We can then call the acceptCall method in TelnyxClient from our ViewModel:
 
+### Handling Multiple Calls
+The Telnyx WebRTC SDK allows for multiple calls to be handled at once. In order to handle multiple calls, you can use the callId to differentiate between calls.
```

Comment:
In order to handle multiple calls => You can use the callId to differentiate the calls.